api/src/main/java/ai/djl/ndarray/NDManager.java

```diff
@@ -1061,6 +1061,54 @@ default NDArray linspace(float start, float stop, int num, boolean endpoint, Dev
         return newSubManager(device).linspace(start, stop, num, endpoint);
     }
 
+    /**
+     * Return random integers from low (inclusive) to high (exclusive).
+     *
+     * @param low Lowest (signed) integers to be drawn from the distribution
+     * @param high one above the largest (signed) integer to be drawn from the distribution
+     * @param shape the {@link Shape} of the {@link NDArray}
+     * @return the drawn samples {@link NDArray}
+     */
+    default NDArray randomInt(int low, int high, Shape shape) {
+        return randomInt(low, high, shape, DataType.INT32);
+    }
+
+    /**
+     * Return random integers from low (inclusive) to high (exclusive).
+     *
+     * @param low Lowest (signed) integers to be drawn from the distribution
+     * @param high one above the largest (signed) integer to be drawn from the distribution
+     * @param shape the {@link Shape} of the {@link NDArray}
+     * @param dataType the {@link DataType} of the {@link NDArray}
+     * @return the drawn samples {@link NDArray}
+     */
+    default NDArray randomInt(int low, int high, Shape shape, DataType dataType) {
```

Comment:
If randomInt can take dataType as input, we should not provide randomLong,
I think we only need:
NDArray randomInteger(long low, long high, Shape shape, DataType dataType);